StoryTowerWeb/schemas/mutations.js

```diff
@@ -1,4 +1,7 @@
-const { StoryCatalog } = require('../models');
+const { StoryCatalog, User } = require('../models');
+const jwt = require('jsonwebtoken');
+const secretKey = process.env.JWT_SECRET || 'non-production-secret'; // Use a default if not set
+const { signToken } = require('../utils/auth.js');
 
 const addScrapedDataToCatalog = async ({ scrapedData }) => {
   console.log('GraphQL mutation for adding scraped data to catalog is executed!');
```

Comment:
should this be a `console.debug`?

https://developer.mozilla.org/en-US/docs/Web/API/console/debug

Reply:
We can try that!